js/pages/samples.js

```diff
@@ -234,12 +234,12 @@ export const renderSamplesPage = async () => {
                     <div class="row" style="width:100%">
                         <div class="consentHeadersFont" style="color:#606060;width:100%">
                             <div>
-                                How Do I Donate My Saliva Sample?
+                                How Do I Donate My MouthwashSample?
```

Comment:
Should there be a space between Mouthwash and Sample?

Reply:
No Gene, there no space needed.

Reply:
> Should there be a space between Mouthwash and Sample?

I believe so.

Reply:
Yes, a space is required between Mouthwash and sample. Adding the space, thank you.